### ISSUE TITLE
profiles - what are they?

### ISSUE THREAD

**Issue by bradbulger** [backdrop-contrib/simple_access_grants#16]
Do you understand what the profile part of the module is for - what it does, how to use it, who would use it? There are several issues for the Drupal module asking these same questions, they are all closed with collections of words that don't add up to anything I can comprehend. Any light that could be shed on this would be helpful.

**Comment by Graham-72** [comment]
When developing this simplified module for Backdrop I came to the conclusion that providing for profiles was an unnecessary complication and I removed this functionality from release 1.x-1.1.0. Nor is it in the more recent 1.x-1.1.1 release.

If I remember correctly the intention was to be able to define more than one access grant as a 'profile' that could then be applied to node or node types in one admin operation. Without profiles one can apply the access grant to each content type individually. 

Each content type now has an administrative entry 'Access settings' where admin can specify which of the user roles can view, update, or delete that type of content. Without profiles it is necessary to set this separately for each content type requiring an access restriction.

I hope this provides some clarification. It is a long time since I used the 'Simple Access' module rather than this one and I may not be remembering it correctly.
